Content.Client/Store/Ui/StoreBoundUserInterface.cs

```diff
@@ -48,6 +51,12 @@ protected override void Open()
         {
             SendMessage(new StoreRequestUpdateInterfaceMessage());
         };
+
+        _menu.SearchTextUpdated += (_, search) =>
+        {
+            _search = search.Trim().ToLowerInvariant();
+            SendMessage(new StoreRequestUpdateInterfaceMessage());
+        };
```

Comment:
this shouldn't be sending messages back to the server. if the code doesn't already, store the current listings on the UI and then filter through them on the client with some function.

Reply:
Yeah, looking through the Uplink UI code, it's definitely in need of a refactor with the way listings are generated. I'll do a little reworking!

Reply:
Done.